python/gui/auto_generated/processing/qgsprocessingenummodelerwidget.sip.in

```diff
@@ -0,0 +1,85 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/processing/qgsprocessingenummodelerwidget.h                  *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
```

Comment:
We could "#define SIP_NO_FILE" these two -- we don't have to expose them

Reply:
Done